x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/details/index.tsx

```diff
@@ -145,37 +146,17 @@ const StyledMinHeightTabContainer = styled.div`
 
 export enum RuleDetailTabs {
   alerts = 'alerts',
-  exceptions = 'exceptions',
+  exceptions = 'ruleExceptions',
   executionResults = 'executionResults',
   executionEvents = 'executionEvents',
```

Comment:
Nit: most Kibana routes are in snake case, should we also name these tabs in snake case for consistency?

Reply:
I was going off of how Hosts does it, but I do agree it's a bit funky. I'm happy to update to snake case.

Reply:
Updated!